config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+- name: sig-auth-serviceaccount-admission-controller-migration
+  annotations:
+    testgrid-dashboards: sig-auth-gce
+    testgrid-tab-name: upgrade-tests
+    # TODO: use sig-auth mailing list
+    testgrid-alert-email: zshihang@google.com
+    testgrid-num-failures-to-alert: "3"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  decorate: true
+  # TODO: lower to once a day
+  interval: 2h
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201014-d8fda77-master
+        args:
+        - --timeout=920
+        - --bare
+        - --scenario=kubernetes_e2e
+        - --
+        - --check-leaked-resources
+        - --check-version-skew=false
+        - --env=STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf
```

Comment:
This isn't consumed by kubetest, you would want to set this as an `env:` on the container instead

Reply:
done in the followup